Source/DafnyCore/Compilers/Python/PythonBackend.cs

```diff
@@ -61,6 +61,14 @@ public class PythonBackend : ExecutableBackend {
     return true;
   }
 
+  public override void CleanSourceDirectory(string sourceDirectory) {
```

Comment:
Shouldn't this just delete the whole `sourceDirectory`, just like the `JavaBackend`?